README.md

```diff
@@ -1,8 +1,140 @@
-# Generic Iterator Project Plan 
+# Case Annotation and Review Tool (CART)
 
-## IDE Set Up
+## What is CART?
 
-### Source Directories
+CART is a module for the 3D slicer program designed to help you manage  iterative analyses, allowing you to focus on implementing and running task you set out to do. Currently, it provides the following capabilities:
+
+### What Does CART Do?
+
+* Managing sequential cases (be they patients, sub-studies, or other collections of data).
+* Cacheing and memory management.
+* User tracking.
+
+A number of features are currently in progress as well, and will be available upon CART's full release:
+
+* Custom task creation and registration.
+* Case pre-fetching/deferred loading.
+* Per-user + per-task configurations.
+
+## Setting up CART
+
+### Prerequisites
+
+- Slicer v5.8 (other versions might work, without guarantee)
+
+### Installation
+
+Clone this repository somewhere can easily access it. You can do this one of two ways:
+
+1. Downloading the repository from GitHub:
+   1. Open the [CART](https://github.com/SomeoneInParticular/CART) repository in your browser.
+   2. Click the green "<> Code" drop-down button on the top-right of of the page
+   <img width="948" height="144" alt="image" src="https://github.com/user-attachments/assets/9e98c9b5-7039-4e1a-bea8-a97c63ee73bd" />
+
+   3. Select "Download ZIP"
+   <img width="432" height="349" alt="image" src="https://github.com/user-attachments/assets/0a1360dd-89fe-4cbb-8e46-c5be4282e5aa" />
+
+   4. Choose where you want the resulting file to be placed in the resulting popup
+   5. Wait for the download to complete
+   6. Once complete, navigate to the `CART-main.zip` file and unzip it (on most OS systems, double-clicking on the file should tell you how to do this)
+
+2. Cloning the repository via `git`:
+   1. Open a terminal in a directory of your choice
+   2. Run the following command to clone the current `CART` repository:
+    ~~~
+    git clone git@github.com:SomeoneInParticular/CART.git
```

Comment:
To unify the clone and `.zip` instructions (and to make navigating to `CART/CART/CART.py` easier), we could instruct users to clone `CART` to `CART-main`?

```suggestion
    git clone git@github.com:SomeoneInParticular/CART.git CART-main
```